pkg/kubelet/dockertools/manager.go

```diff
@@ -578,8 +578,8 @@ func (dm *DockerManager) runContainer(
 			labels[kubernetesContainerLabel] = container.Name
 		}
 	}
-	memoryLimit := container.Resources.Limits.Memory().Value()
-	cpuShares := milliCPUToShares(container.Resources.Limits.Cpu().MilliValue())
+	memoryLimit := container.Resources.Requests.Memory().Value()
+	cpuShares := milliCPUToShares(container.Resources.Requests.Cpu().MilliValue())
```

Comment:
I wish we had a test that validates this in the resulting docker.Config.

Could then evolve it when we add hard caps.